### PR TITLE
Testing To Do list

### PR DESCRIPTION
**Pull request by Atifzada**
### In this PR we have:

- Used jest framework for testing.
- Created a test file(s) `([..].test.js) `for a file(s) containing the following functions:

1. a function for editing the task description.
2. a function for updating an item's 'completed' status.
3. the "Clear all completed" function.

- Used the mock storage object created in Part 1 of this project to mock the localStorage updates.
- Mock the HTML elements to test DOM manipulation functions.
- Make sure you group your tests using the describe`() method`.

### CODE REVIEW

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
Hi @Atifzada 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌ ❌ ❌ 

- [x] You have submitted a project, that does not have a Readme file. 🤷‍♂️

- [x] kindly add a README file to your project using the provided [Microvese README template](https://github.com/microverseinc/readme-template) and customize its content according to your project. 👍.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Atifzada** [comment]
Thank You for the review. READ ME is added now

File: README.md [hunk 1]
```diff
@@ -34,7 +34,8 @@
 
 # 📖 To-do-list <a name="about-project"></a>
```
Comment (Yerimah) [inline README.md:35]: - [OPTIONAL ] Always make your README file as descriptive as possible. I would suggest that you add detailed instructions on how to run test in the Run test section in this format. 
- Install Jest as a development dependency for npm `npm install --save-dev jest` and 
- Run `npm run test` in the command line

**Review by Yerimah — APPROVED** [review]
# STATUS: APPROVED 🔥 

Hi @Team, 👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

68747470733a2f2f36342e6d656469612e74756d626c722e636f6d2f38376530623634363963393537626638393663646563316361623634366163662f74756d626c725f696e6c696e655f706b316264616b434370317270637239615f3530302e676966

## Highlights
- Meaningful commits ✔️ 
- Descriptive PR title ✔️ 
- Decriptive PR summary ✔️ 
- No linter error ✔️ 
- All tests are passing ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**